source/NVDAObjects/IAccessible/ia2Web.py

```diff
@@ -26,6 +26,20 @@ class Ia2Web(IAccessible):
 	IAccessibleTableUsesTableCellIndexAttrib=True
 	caretMovementDetectionUsesEvents = False
 
+	def isDescendantOf(self, obj):

```

Comment:
add type annotations

```suggestion
	def isDescendantOf(self, obj: "NVDAObjects.NVDAObject") -> bool:
```